lib/tasks/config.rake

```diff
@@ -2,7 +2,7 @@ require 'optparse'
 require 'json'
 require 'yaml'
 
-desc "Configure Foreman in-database settings, see rake config -- --help for more details"
+desc "Configure Foreman in-database settings, see rake -- config --help for more details"
```

Comment:
`foreman-rake config` only makes sense if using `foreman-rake` in a production case. Since this is the rake task itself, we'd want this to reflect how to use the rake task, no?